src/stellar/stellarEngine.ts

```diff
@@ -373,7 +372,7 @@ export class StellarEngine extends CurrencyEngine<StellarPlugin> {
     // Check if destination address is activated
     let mustCreateAccount = false
     const activated = this.activatedAccountsCache[publicAddress]
-    if (activated === false) {
+    if (!activated) {
```

Comment:
This shouldn't be changed. If activated is undefined we don't test the second conditional

Reply:
This was caused by an eslint autoformat. For some reason eslint won't let you use `activated === false` since it thinks that `activated` can never be undefined. I re-ordered the IF statement to first check undefined then for false.